draft-davidson-pp-protocol.md

```diff
@@ -163,6 +168,49 @@ We assume that all protocol messages are encoded into raw byte format
 before being sent. We use the TLS presentation language {{RFC8446}} to
 describe the structure of protocol data types and messages.
 
+# Background
+
+We discuss the core motivation behind the protocol along with the
+guarantees and assumptions that we make in this document.
+
+## Motivating use-cases
+
+The Privacy Pass protocol was originally developed to provide anonymous
+authorization of users of anonymity-preserving tools on the Internet,
+such as Tor and VPNs. In particular, the protocol allows clients to
+reveal authorization tokens that they have been issued without linking
+the authorization to the actual issuance event. This means that the
+tokens cannot be used to link the browsing patterns of clients that
+reveal tokens.
+
+The web performance company Cloudflare uses a form of Privacy Pass to
+issue tokens to users that have completed Internet challenges
+{{DGSTV18}}. This allows clients to redeem tokens instead of solving
+additional challenges in the future. Other use-cases have since arisen
+with similar core security goals: {{TrustTokenAPI}}, {{PrivateStorage}},
+{{OpenPrivacy}}, {{Brave}}.
+
+## Anonymity and security guarantees
+
+As mentioned above, the protocol provides anonymity-preserving tokens
+for authorizing clients. Throughout this document, we use the terms
+"anonymous", "anonymous-preserving" and "anonymity" to refer to the core
+security guarantee of the protocol. This guarantee is the following
+statement.
+
+- Any client that is issued a token by a server key and subsequently
+  redeems it, remains anonymous within the set of all clients issued
+  tokens under the same key.
+
+We also require a security guarantee for the server to maintain the
+utility of the authorization protocol.
```

Comment:
```suggestion
We also require that clients cannot forge tokens, as otherwise Privacy Pass would
have little value as an authorization protocol. This requirement is stated as follows.
```